stdlib/2and3/threading.pyi

```diff
@@ -145,17 +145,7 @@ class Semaphore:
         def acquire(self, blocking: bool = ...) -> bool: ...
     def release(self) -> None: ...
 
-class BoundedSemaphore:
-    def __init__(self, value: int = ...) -> None: ...
-    def __enter__(self) -> bool: ...
-    def __exit__(self, exc_type: Optional[Type[BaseException]],
-                 exc_val: Optional[BaseException],
-                 exc_tb: Optional[TracebackType]) -> Optional[bool]: ...
-    if sys.version_info >= (3,):
-        def acquire(self, blocking: bool = ..., timeout: float = ...) -> bool: ...
-    else:
-        def acquire(self, blocking: bool = ...) -> bool: ...
-    def release(self) -> None: ...
+class BoundedSemaphore(Semaphore): ...
```

Comment:
Strictly speaking in Python 2 `BoundedSemaphore` is a function that returns an instance of the `_BoundedSemaphore` class, for some reason, but in 3.9 at least it is in fact a class. I don't think we need to worry about matching 2.7 precisely here unless stubtest would get upset.

Reply:
stubtest doesn't work on Python 2.7 :-)